dev_simulate_test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -x
```

Comment:
what's this file about?

Reply:
sorry. It was a file i was using to understand git behavior. Not needed.